src/main/java/Task.java

```diff
@@ -0,0 +1,74 @@
+//Create a task class
+public class Task {
+    protected String taskName;
+    protected boolean isDone;
+    public Task(String taskName){
+        this.taskName = taskName;
+        this.isDone = false;
+    }
+    //method to mark as done
+    public void markAsDone(){
+        this.isDone = true;
+    }
+    //method to mark as not done
+    public void markAsNotDone(){
+        this.isDone = false;
+    }
+    //toString method to print the status of the task followed by the task name
+    public String toString(){
+        if(this.isDone){
+            return "[X] " + this.taskName;
+        }else{
+            return "[ ] " + this.taskName;
+        }
+    }
+}
+
+//Create a todo class that inherits from the task class
+class Todo extends Task {
+    public Todo(String taskName){
+        super(taskName);
+    }
+    //toString method to print the status of the task followed by the task name
+    public String toString(){
```

Comment:
```suggestion
    @Override
    public String toString(){
```
Add the decorator to add more semantic meaning that this method is overriding the parent class method.